File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1541_basico.js
import { question } from "readline-sync";

function main() {
    let entrada = get_Text().split(` `);
    let a = parseInt(entrada[0]);
    let b = parseInt(entrada[1]);
    let c = parseInt(entrada[2]);

    while(a !== 0) {
        let areaTotal = a * b;
        let areaMaximaTotal = areaTotal / c;
        let lado =(Math.sqrt(areaMaximaTotal));
        print(`${parseInt(lado * 10)}`);

        entrada = get_Text().split(` `);
        a = parseInt(entrada[0]);
        b = parseInt(entrada[1]);
        c = parseInt(entrada[2]);

    }

}

function get_Text() {
    return question();
}

function print(texto) {
    return console.log(texto);
}

main();

File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1541_basico.py
import math

def main():
    entrada = get_Text().split(' ')

    while len(entrada) == 3:
        a = int(entrada[0])
        b = int(entrada[1])
        c = int(entrada[2])

        area_Total = a * b
        area_Maxima_Total = area_Total / c
        lado = math.sqrt(area_Maxima_Total)
        print(f'{int(lado * 10)}')

        entrada = get_Text().split(' ')
    
def get_Text():
    return input().strip()

main()